fe/fe-core/src/main/java/com/starrocks/system/SystemInfoService.java

```diff
@@ -627,6 +774,20 @@ public long saveBackends(DataOutputStream dos, long checksum) throws IOException
         return checksum;
     }
 
+    public long saveComputeNodes(DataOutputStream dos, long checksum) throws IOException {
+        ImmutableMap<Long, ComputeNode> idToComputeNode = idToComputeNodeRef;
```

Comment:
You should make sure that persist meta/program is compatible with old program/meta.

Reply:
this has changed